client/src/ui/molecules/pagination/index.scss

```diff
@@ -0,0 +1,16 @@
+@import "~@mdn/minimalist/sass/vars/layout";
+
+.pagination {
+  @media #{$mq-small-desktop-and-up} {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  li {
+    display: inline;
+
+    a {
+      display: inline-block;
+    }
+  }
+}
```

Comment:
I think this should be put into the `search-results.scss`. It's overkill to have its own file if the `<Pagination>` component doesn't deserve to be its own file.